src/openforms/conf/base.py

```diff
@@ -1119,6 +1119,7 @@
 # Django Modeltranslation
 #
 MODELTRANSLATION_DEFAULT_LANGUAGE = "nl"
+MODELTRANSLATION_ENABLE_FALLBACKS = False
```

Comment:
@sergei-maertens I'm not sure if there is a scenario in which we do want to fallback to another language?

Reply:
I think it's realistic and falling back to NL as a default is sane behaviour. Perhaps https://django-modeltranslation.readthedocs.io/en/latest/usage.html#fallback-undefined is a better, more targeted approach to not treat empty string for those literal fields as a missing translation?

Using a sentinel `empty = object()` seem like valid options here (since `None` would be the non-default field empty value as mt-fields are nullable)